### PR TITLE
Fix typo in metadata (eBPF)

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Fix typo in eBPF metadata

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: collectors/ebpf.plugin/integrations/ebpf_processes.md [hunk 1]
```diff
@@ -87,7 +87,7 @@ Metrics:
 |:------|:----------|:----|
 | app.process_create | calls | calls/s |
 | app.thread_create | call | calls/s |
-| app.task_exit | callp | calls/s |
```
Comment (ilyam8) [inline collectors/ebpf.plugin/integrations/ebpf_processes.md:90]: undo this change, the docs gen script will update this file

**Review by ilyam8 — APPROVED** [review]

File: collectors/ebpf.plugin/integrations/ebpf_processes.md (hunk 1 above)
Comment (thiagoftsm) [reply collectors/ebpf.plugin/integrations/ebpf_processes.md]: Done!

**Review by ilyam8 — APPROVED** [review]

**Review by Dim-P — APPROVED** [review]

**Review by stelfrag — APPROVED** [review]
